.circleci/create_circleci_config.py

```diff
@@ -326,7 +326,7 @@ def job_name(self):
     "examples_tensorflow",
     cache_name="tensorflow_examples",
     docker_image=[{"image":"huggingface/transformers-examples-tf"}],
-    install_steps=["uv venv && uv pip install ."],
+    install_steps=["uv venv && uv pip install . && uv pip install -r examples/tensorflow/_tests_requirements.txt"],
```

Comment:
Mmmm again, that is not what we should do. The CI is gonna get slower because we just need the docker that is built to run this.